core/api/src/main/java/org/eclipse/sensinact/core/whiteboard/WhiteboardResourceDescription.java

```diff
@@ -0,0 +1,34 @@
+/*********************************************************************
+* Copyright (c) 2024 Kentyou.
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which is available at https://www.eclipse.org/legal/epl-2.0/
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*   Thomas Calmant (Kentyou) - initial implementation
+**********************************************************************/
+package org.eclipse.sensinact.core.whiteboard;
+
+import java.time.Duration;
+
+/**
+ * Interface that an auto-descriptive value resource handler must implement if
+ * it expects the whiteboard to create its resource automatically
+ *
+ * @param <T> Type of resource
+ */
+public interface WhiteboardResourceDescription<T> {
```

Comment:
Using 
```suggestion
public interface WhiteboardResourceDescription<T> extends WhiteboardGet<T> {
```
Would force the reified type of the the description and the get to be the same, or we could use an indirection to force the type variable of the description to be a super type of the type variable for the get.